docs/lcdproc-user/how-to-obtain.docbook

```diff
@@ -25,8 +25,8 @@ around on the internet:
   <term>Older releases</term>
   <listitem>
     <para>
-    Older releases of LCDproc can be found on the Sourceforge project page at
-    <ulink url="http://sourceforge.net/projects/lcdproc/files/lcdproc/"></ulink>.
+    Older releases of LCDproc can be found on the GitHub project page at
+    <ulink url="https://github.com/lcdproc/lcdproc"></ulink>.
     </para>
```

Comment:
I'll look into how we can host tarballs of older releases on Github so we can leave the statement there (pointing to Github).